integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/SetDeviceCommunicationSettings.feature

```diff
@@ -16,11 +16,12 @@ Feature: SmartMetering Management - Set Device Communication Settings
       | IpAddressIsStatic        | false             |
       | UseSn                    | false             |
       | UseHdlc                  | true              |
+      | Polyphase                | true              |
```

Comment:
Is there a test with Polyphase false as well?

Reply:
Yes, i also added default values as well